src/lib/cyleaflet_clientside.js

```diff
@@ -87,4 +95,43 @@ window.dash_clientside.cyleaflet = {
             return e;
         });
     },
+    updateLonLat: function (elements) {
+        if (elements.length > 0) {
+            return elements.map((e) => {
+                if (
+                    typeof e.position !== 'undefined' &&
+                    Object.prototype.hasOwnProperty.call(e.position, 'x') &&
+                    e.position.x !== 0 &&
```

Comment:
But what if the node is really located at position (0, 0)?

Reply:
I haven't encountered that issue since cytoscape positions are calculated with the `lonLatToXY` function and for `latitude=0` the resulting `y` is `7.081154550627917e-10`. Which is pretty close to zero but not zero ;)